docs/data/data-grid/filtering/quick-filter.md

```diff
@@ -122,6 +122,25 @@ In the following demo, the quick filter value `"Saint Martin, Saint Lucia"` will
 
 {{"demo": "QuickFilteringCustomizedGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
+## Ignore diacritics (accents)
```

Comment:
I wanted to keep this relatively high in the filter docs so developers won't miss it without digging deep into customizations.
I made an entry point for this for every filter type:
- https://deploy-preview-10569--material-ui-x.netlify.app/x/react-data-grid/filtering/#ignore-diacritics-accents
- https://deploy-preview-10569--material-ui-x.netlify.app/x/react-data-grid/filtering/quick-filter/#ignore-diacritics-accents
- https://deploy-preview-10569--material-ui-x.netlify.app/x/react-data-grid/filtering/header-filters/#ignore-diacritics-accents

The demo is in the quick filter docs because it's easier to demonstrate how it works with quick filtering.